utilities/_layout.scss

```diff
@@ -0,0 +1,36 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+
+
+.grid {
+    @include grid(repeat(6, 1fr), 16px, 24px);
+    
+    @media ($mobile-upper <= width <= $desktop-lower) {
+        @include grid(repeat(12, 1fr), 32px, 75px);
+    }
+    
+    @media (width >= ($desktop-lower + 1)) {
+        @include grid(repeat(12, 1fr), 32px, 100px);
+    }
+}
+
```

Comment:
Your codepen has some grid-item styles but I can't see them here?

Reply:
The grid-items are just to show the object area in the demo, not necessary for the this